mlrun/runtimes/serving.py

```diff
@@ -322,11 +322,17 @@ def set_tracking(
 
         # Applying model monitoring configurations
         self.spec.track_models = True
-        self.spec.tracking_policy = model_monitoring.TrackingPolicy()
+        self.spec.tracking_policy = ""
         if tracking_policy:
-            self.spec.tracking_policy = self.spec.tracking_policy.from_dict(
-                tracking_policy
-            )
+            if isinstance(tracking_policy, dict):
+                # Convert tracking policy dictionary into `model_monitoring.TrackingPolicy` object
+                self.spec.tracking_policy = model_monitoring.TrackingPolicy().from_dict(
```

Comment:
Pretty sure this is a class method.
```suggestion
                self.spec.tracking_policy = model_monitoring.TrackingPolicy.from_dict(
```

Reply:
I beat you to it :-)